templates/webshop/search-results.jinja

```diff
@@ -8,8 +8,37 @@
 {% endblock breadcrumb %}
 
 {% block main %}
+
 <section>
   <div class="container">
+  {% if facets %}
+  {% set facets = facets | add_display_counts %}
+  {% block sidebar %}
+  <div class="col-md-3">
+    <div class="sidebar">
+      <h4>{% trans %}Refine your search{% endtrans %}</h4>
+      <form id="filter-form-sidebar">
+        <input type="hidden" name="q" value="{{ request.args.get('q') }}" />
+        <input type="hidden" name="page" value="1"/>
+        {% for key, value in facets.iteritems() %}
+        <ul>
+        <lh>{{ key }}</lh>
+        {% set facet_checked = request.args.getlist(key) %}
+        {% for term in value['terms'] %}
+          <li>
+            <input type="checkbox" name="{{ key }}" value="{{ term['term'] }}"
```

Comment:
I think you should move this logic to a macro with a block inside for inheritance.

```jinja
macro {
  macro_block {
    ...
  }
}
```